.rules/.markdown-lint.yml

```diff
@@ -0,0 +1,36 @@
+---
+###########################
+###########################
+## Markdown Linter rules ##
+###########################
+###########################
+
+# Linter rules doc:
+# - https://github.com/DavidAnson/markdownlint
+#
+# Note:
+# To comment out a single error:
+#   <!-- markdownlint-disable -->
+#   any violations you want
+#   <!-- markdownlint-restore -->
+#
+
+###############
+# Rules by id #
+###############
+MD004: false                  # Unordered list style
+MD007:
+  indent: 2                   # Unordered list indentation
+MD013:
+  line_length: 808            # Line length
```

Comment:
Why we want a 808 line length?

Reply:
I copy pasted the default `.markdown-lint.yml` file from super-linter and i just added the 

```yaml
 MD041: false 
```

Because we have markdown files does not start with a top level header `#` instead `###`. (Release notes markdown files)

If we want to have a small line width, feel free to submit a PR to our "policeman"